composer.json

```diff
@@ -41,11 +41,13 @@
         }
     },
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "ext-iconv": "*",
+        "ext-mbstring": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4",
-        "friendsofphp/php-cs-fixer": "2.2.*"
+        "phpunit/phpunit": "^4.8",
+        "friendsofphp/php-cs-fixer": "^2.2"
```

Comment:
Punic requires PHP-CS-Fixer version 2.2, not PHP-CS-Fixer "compatible" with version 2.2.
That's because there are incompatible changes between 2.2 and 2.11: for instance, the @Symfony fixer set does not include the Yoda style in 2.2, but it does in 2.11.
And in none of my projects I will ever switch to Yoda style... The reason why Yoda style exists is to avoid problems like having assignments instead of equality checks (`if ($a = 0) {} ` vs ``if ($a == 0) {}`), but in my personal opinion making the code harder to be read is not the solution: the solution is to add PHPUnit tests.
For instance, would you me better understand if speak I this like? :wink: